adapters/MyTarget/MyTargetAdapter/GADMAdapterMyTargetExtras.m

```diff
@@ -24,4 +26,24 @@ - (nonnull instancetype)init {
   return self;
 }
 
+- (void)setParameter:(nullable NSString *)parameter forKey:(nonnull NSString *)key {
```

Comment:
Question: Is there a list that shows what parameters are supported?

Reply:
Actually there is no such list. We allow users to set [String:String] dictionary as custom parameters with no additional requirements to Key string.